pkgs/development/python-modules/pywebview/default.nix

```diff
@@ -17,18 +19,18 @@
 buildPythonPackage rec {
   pname = "pywebview";
   version = "4.3.3";
-  format = "setuptools";
+  format = "pyproject";
 
   disabled = pythonOlder "3.5";
 
-  src = fetchFromGitHub {
-    owner = "r0x0r";
-    repo = "pywebview";
-    rev = "refs/tags/${version}";
-    hash = "sha256-8BkbO7C8cYDIQWWCKaXFjfD45L5KVG1tDZJl+uW5g9g=";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-BI6JyYKESAGOsn1Q1G+0p75M3AJOTaz48DU72hfqSoE=";
```

Comment:
Do we need this change?

Reply:
Yes. `setuptools-scm` needs some metadata that GitHub's source archives don't provide. The error you would get otherwise tells you to either clone the Git repo or download a source archive from PyPI.

Reply:
Can we use `env.SETUPTOOLS_SCM_PRETEND_VERSION = version;` instead?

Reply:
done!